integration/oasis/haversine/distance.go

```diff
@@ -0,0 +1,29 @@
+package haversine
```

Comment:
@wangyoucao577 
Still investigate on which geo lib to import.
This sub-package will be removed later.